src/command.py

```diff
@@ -61,6 +61,22 @@ def is_rank_command(body: Dict[str, Any]) -> bool:
     return False
 
 
+def is_opt_out_command(body: Dict[str, Any]) -> bool:
+    """Check if the command is for opting out"""
+    for option in body["data"]["options"]:
```

Comment:
I am okay with this but could also be 

```python
return any(option for option in body["data"]["options"] if option.get("name") == "opt-out")
```


something like that?

Reply:
Yes, using this could make the code more 'pythonic'. I stuck with this way because we already had a similar function to check the rank command. I wanted to keep things consistent without refactoring the existing part!